OptimizelySDK/OptimizelyUserContext.cs

```diff
@@ -0,0 +1,130 @@
+﻿/* 
```

Comment:
Please add separate class and test cases for optimizelyusercontext.
including setAttributes and using multiple threads.